plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -354,13 +360,13 @@ def count_error(self, error):
             return False  # Don't log
         self.error_count += 1
         if _is_retryable(error) and self.error_count <= self.retries:
-            logger.warning(
-                'Retry attempt %s of %s after error: %s',
+            logger.debug(
```

Comment:
I think we want to move `self.error_count` to down below the `else` statement (or similar). I think the way this is implemented, a `retries` value of 1 will in fact not retry queries that fail with server errors!

Reply:
and I think the `self.error-count-1` below is wrong here too - it will show one fewer attempt than was actually made (at least with the logic as it exists currently). Just looking at this further - do we want to use `repr()` to log the error? Or should that be `str()` instead?

Reply:
ignore all of this - I was way off, sorry